manager/assets/modext/widgets/core/modx.grid.js

```diff
@@ -129,7 +130,8 @@ MODx.grid.Grid = function(config) {
             }
 
             config.cm.columns.push({
-                width: config.actionsColumnWidth || defaultActionsColumnWidth
+                id: config.cm.columns.length
```

Comment:
```suggestion
               id: 'modx-actions'
```

Reply:
Couple things (note, I'm not looking to squabble about this, I'm just a stickler on details and am curious)—
- Where is the regex you refer to and how do you see it fail (i.e., where do you see the effects)? It's actually kind of an odd rule; why would one be looking specifically for one or more of a backslash or the letter "s"?
- Regardless of the answer to that, the id 'modx-actions' does work just fine in the end—causes no errors and the classes get added to the header and column cells as intended.

In any case, if you feel strongly about doing it the way you did that's not a big deal - I just wanted to throw this out there, especially if it might be desirable to _easily_ style this column in a particular way in the future.

Reply:
BTW, whatever you decide, if you see the "Resolve Conversation" button for these comments I've made go ahead and close 'em out. I don't see the button for some reason.

Reply:
bah, I had an issue with double escaping while testing the regex, you were right it works. adjusted it.

for your reference, the regex is in https://docs.sencha.com/extjs/3.4.0/source/GridView.html#Ext-grid-GridView
search for: `this.colRe = new RegExp('x-grid3-td-([^\\s]+)', '');`